openshift/templates/api/api-deploy.yaml

```diff
@@ -230,6 +231,11 @@ objects:
                       name: ${NAME}
                       key: encryption-key
 
+                - name: EFILING_ENABLED
```

Comment:
Since this setting does not contain any sensitive information there is no need to store it as a secret.  You can therefore switch this to be a regular environment variable like `APP_CONFIG`.